sys/include/net/ng_fib.h

```diff
@@ -122,6 +130,27 @@ int fib_get_next_hop(kernel_pid_t *iface_id,
                      uint8_t *dst, size_t dst_size, uint32_t dst_flags);
 
 /**
+* @brief provides a set of destination addresses matching the bits specified with
+* the provided bitmap and type.
+* If the out buffer is insufficient low, the function will continue to count the
```

Comment:
could you also add the fact, that the buffer is allowed to be null? Something like:
>If the out buffer is insufficient low or null, ...